docs/source/resources/usage_tips/glossary.rst

```diff
@@ -8,7 +8,7 @@ Glossary
     :sorted:
 
     feature
-        A transformation of data used for machine learning.  featuretools has a custom language for defining features as described :ref:`here <primitives>`. All features are represented by subclasses of :class:`FeatureBase`.
+        A transformation of data used for machine learning.  featuretools has a custom language for defining features as described :doc:`here </getting_started/primitives>`. All features are represented by subclasses of :class:`FeatureBase`.
```

Comment:
since we no longer have custom anchors in these notebook docs, I figured it was best to switch to using `:doc:` here. The alternative would be to add the anchors via rst cells if there's some difference between `:doc:` and `:ref:` that makes using `:ref:` preferable.

Reply:
One benefit I see from anchors is that if we moved the document, we wouldn't need to update the `:ref:` link

Reply:
updated to use anchors and `:ref:`